PKHeX/Saves/SaveFile.cs

```diff
@@ -8,6 +8,7 @@ public abstract class SaveFile
     {
         public static bool SetUpdateDex = true;
         public static bool SetUpdatePKM = true;
+        public static bool ChkIllegal = true;
```

Comment:
This variable is set but never used.  It (and the things setting it) can be safely removed.